apps/v1/types.go

```diff
@@ -14,9 +14,9 @@ import (
 // +genclient:method=GetScale,verb=get,subresource=scale,result=k8s.io/api/extensions/v1beta1.Scale
 // +genclient:method=UpdateScale,verb=update,subresource=scale,input=k8s.io/api/extensions/v1beta1.Scale,result=k8s.io/api/extensions/v1beta1.Scale
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
-// +k8s:prerelease-lifecycle-gen:introduced=3.0
-// +k8s:prerelease-lifecycle-gen:deprecated=4.14
-// +k8s:prerelease-lifecycle-gen:removed=4.10000
+// +k8s:prerelease-lifecycle-gen:introduced=1.1
+// +k8s:prerelease-lifecycle-gen:deprecated=1.27
```

Comment:
Note: the versioning was originally intentionally set to follow the openshift versioning

Reply:
Understood, but we are making use of upstream functionality that only truly understands kubernetes versions.